index.html

```diff
@@ -0,0 +1,157 @@
+                <button>html</button>
+                <button>bootstrap</button>
+                <button>Ruby</button>
+            </div>
+        </div>
+        <div class="seeall">
+            <button>See Project</button>
+        </div>
+        <div class="work-items work-items-project">
+            <h5>Profesional Art Printing Data</h5>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+            <div class="work-items-project-button">
+                <button>html</button>
+                <button>bootstrap</button>
+                <button>Ruby</button>
+            </div>
+        </div>
+        <div class="seeall">
+            <button id="btnSee">See Project</button>
+        </div>
+        <div class="work-items work-items-project">
+            <h5>Profesional Art Printing Data</h5>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+            <div class="work-items-project-button">
+                <button>html</button>
+                <button>bootstrap</button>
+                <button>Ruby</button>
+            </div>
+        </div>
+        <div class="seeall">
+            <button>See Project</button>
+        </div>
+        <div class="work-items"></div>
+        <div class="work-items"></div>
+        <div class="work-items"></div>
+        <div class="work-items"></div>
+        <div class="work-items"></div>
+
+    </section>
+    <section class="about">
+        <div class="about-items">
+            <h2>About me</h2>
+        </div>
+        <div class="about-items">
+            <p>
+                Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+            </p>
+        </div>
+        <div class="about-items">
+            <button>
+                Get My Resume
+            </button>
+                
+        </div>
+        <div class="about-items">
+            <img class="background" src="./img/Group.png" alt="">
+        </div>
+        <div class="about-items">
+            <hr>
+        </div>
+        <div class="about-items">
+            <ul>
+                <li>Languages
+                    <br>
+                    Javascript
+                    <br>
+                    Ruby
+                    <br>
+                    Html
+                    <br>
+                    Css
+                </li>
+                <li>Frameworks
+                  <br>
+                  Bootstrap
+                  <br>
+                  Ruby on Rails
+                  <br>
+                  RSPec
+                  <br>
+                  CapyBara
+                  <br>
+                  Selenium
+                </li>
```

Comment:
- [ ] For each category, the technology used (e.g., "JavasScript", "HTML" and "CSS") are also a list `ul` and each of them a `li`